oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java

```diff
@@ -343,6 +345,31 @@ private boolean isMatch() {
                                 break;
                         }
                         break;
+                    case MULTI_INTS:
+                        int[] ivalueArray = ((MultiIntValuesHolder)metrics).getValues();
+                        int[] iaexpected = RunningRule.this.threshold.getIntValuesThreshold();
+                        MULTI_VALUE_CHECK: for (int i = 0; i < ivalueArray.length; i++) {
+                            ivalue = ivalueArray[i];
+                            iexpected = 0;
+                            if (iaexpected.length > i) {
+                                iexpected = iaexpected[i];
+                            }
```

Comment:
Is it reasonable to make it `0` by default? My intuition of the following configs:

```yml
threshold: 1000,1000
```

may be just monitor and alarm the `p99` and `p95` only, ignoring others percentile metrics, and your implementation seems to always send alarm? (because the other metrics may be almostly always > 0).

And are there any possibility when the users want to monitor `p99` and `p75`, how to config the threshold?

Reply:
This is only a fail-safe mechanism, in case user doesn't set enough values.

Reply:
So the users are expected to set all thresholds for all percentile metrics?

Reply:
Yes. Or set to 0, then percentile alarm wouldn't be triggered by it. Because you will use `>` as OP.

Reply:
Sorry, my mistake. All thresholds must be set at current implementation.

Reply:
I haven't found a better way to support setting part of it, maybe set `-` to represent skipping?

Reply:
> I haven't found a better way to support setting part of it, maybe set `-` to represent skipping?

Sounds good to me

Reply:
Done, please review.